gammapy/maps/axes.py

```diff
@@ -2973,6 +2973,56 @@ def to_header(self, format="gadf", idx=0):
 
         return header
 
+    def group_table(self, interval_edges):
+        """Compute bin groups table for the TimeMapAxis, given coarser bin edges.
+
+        Parameters
+        ----------
+        interval_edges : list of `~astropy.time.Time` or `~astropy.units.Quantity`
+            Start and stop time for each interval to compute the LC.
+
+        Returns
+        -------
+        groups : `~astropy.table.Table`
+            Group table.
+        """
+
+        for _, edge in enumerate(interval_edges):
+            if not isinstance(edge, Time):
+                interval_edges[_] = self.reference_time + interval_edges[_]
+
+        time_intervals = list(zip(interval_edges[::2], interval_edges[1::2]))
+        group_table = Table(
+            names=("idx_min", "idx_max", "time_min", "time_max", "bin_type"),
+            dtype=("i8", "i8", "f8", "f8", "S10"),
+        )
+
+        for time_interval in time_intervals:
+            mask1 = self.time_min >= time_interval[0]
+            mask2 = self.time_max <= time_interval[1]
+            mask = mask1 & mask2
+            if np.any(mask):
+                idx_min = np.where(mask)[0][0]
+                idx_max = np.where(mask)[0][-1]
+                time_min = self.time_min[idx_min]
+                time_max = self.time_max[idx_max]
+                bin_type = "normal   "
+            else:
+                idx_min = idx_max = -1
+                time_min = self.time_min[idx_min]
```

Comment:
these 2 lines can be outside the if else block to avoid repetition, no?

Reply:
Modified in the latest commit, thanks!